pkg/broker/filter/filter_handler.go

```diff
@@ -235,10 +235,15 @@ func (h *Handler) handleDispatchToReplyRequest(ctx context.Context, trigger *eve
 	target := broker.Status.Address
 
 	reportArgs := &ReportArgs{
-		ns:          trigger.Namespace,
-		trigger:     trigger.Name,
-		broker:      trigger.Spec.Broker,
-		requestType: "reply_forward",
+		ns:            trigger.Namespace,
+		trigger:       trigger.Name,
+		broker:        trigger.Spec.Broker,
+		requestType:   "reply_forward",
+		requestScheme: request.URL.Scheme,
```

Comment:
Similar to the other comment, this will always be empty as we are on the server side, we need to check if TLS is being used or not some other way. Maybe take a look at the comment from the issue I linked earlier that includes a workaround, or try some other way